docs/topics/kotlin-evolution.md

```diff
@@ -205,4 +205,21 @@ For fully stable versions of the compiler, the default binary compatibility prot
 
 This protocol is designed for comfortable updates as no project can be blocked from updating its dependencies even if it's using a slightly outdated compiler.
 
-Please note that not all target platforms have reached this level of stability (but Kotlin/JVM has).
\ No newline at end of file
+Note that not all target platforms have reached this level of stability, but Kotlin/JVM has.
+
+#### Kotlin klib binaries
+
+Kotlin klib binaries have reached the [Stable](components-stability.md#stability-levels-explained) level in Kotlin 1.9.20.
+However, there are some compatibility details you need to keep in mind:
+
+* klib binaries are backwards compatible starting with Kotlin 1.9.20. For example, the 2.0.x compiler can read binaries
+  produced by the 1.9.2x compiler.
+* Older compilers reject klib binaries that rely on new features. For example, the 1.9.20 compiler rejects binaries that
```

Comment:
Well... I think that we should just remove this phrase because the next one is saying that forward compatibility is not guaranteed at all, and this part in JVM was added for forward compatibility cases

Reply:
OK, let's leave only backward and forward compatibility recommendations then